### PR TITLE
Add images

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: vendor-extra/JatsContentBundle/tests/EventListener/BuildView/GraphicImageListenerTest.php [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+    }
+
+    public function nodeProvider() : iterable
+    {
+        yield 'different namespace' => [
+            <<< XML
+<graphic xmlns="http://example.com" xmlns:xlink="http://www.w3.org/1999/xlink"
+    xlink:href="http://example.com/image.jpg"/>
+XML
+            ,
```
Comment (giorgiosironi) [inline vendor-extra/JatsContentBundle/tests/EventListener/BuildView/GraphicImageListenerTest.php]: random comma mandated by coding standards?

File: vendor-extra/JatsContentBundle/src/EventListener/BuildView/GraphicImageListener.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    protected function canHandleElement(string $element) : bool
+    {
+        return '{http://jats.nlm.nih.gov}graphic' === $element;
+    }
+
+    protected function canHandleArguments(array $arguments) : bool
+    {
+        return !array_has_key($arguments, 'image');
+    }
```
Comment (giorgiosironi) [inline vendor-extra/JatsContentBundle/src/EventListener/BuildView/GraphicImageListener.php]: nothing surprising in the Listeners, though I would have expected to see changes to the XML fixtures in `.docker/api/data` to add examples of these?

File: vendor-extra/JatsContentBundle/tests/EventListener/BuildView/GraphicImageListenerTest.php (hunk 1 above)
Comment (thewilkybarkid) [reply vendor-extra/JatsContentBundle/tests/EventListener/BuildView/GraphicImageListenerTest.php]: PHP 7.3 would make it better (https://wiki.php.net/rfc/flexible_heredoc_nowdoc_syntaxes).

Bit ugly, but 🤷‍♂️.

File: vendor-extra/JatsContentBundle/src/EventListener/BuildView/GraphicImageListener.php (hunk 2 above)
Comment (thewilkybarkid) [reply vendor-extra/JatsContentBundle/src/EventListener/BuildView/GraphicImageListener.php]: Need to update. Don't want to commit/serve an image, but could use a real IIIF one for now.

**Pull request merged**